composer.json

```diff
@@ -7,16 +7,16 @@
   "require": {
     "php": "^7.4 || ^8.0",
     "ext-xsl" : "*",
-    "codeigniter4/framework": "^4.4",
-    "codeigniter4/shield": "^1.0@beta",
-    "league/commonmark": "^2.3",
-    "michalsn/codeigniter-htmx": "^1.2.2",
-    "s9e/text-formatter": "^2.13"
+    "codeigniter4/framework": "4.4.1",
```

Comment:
I don't see the need to bake these to a path release version. I'd rather rely on semantic versioning and allow things to be updated when it should be safe to do so.

Reply:
I removed the commit for it, hope I got it right.